### PR TITLE
Fix 7.4 to 7.5 migration config

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[7.4 to 7.5 database migration script](https://trello.com/c/d4gBVvGP/593-74-to-75-database-migration-script)

File: lib/config-migration.js [hunk 1]
```diff
@@ -407,7 +411,11 @@ function migrateComplianceTriggers (config) {
       createTrigger(requirements.sanctions, global.sanctionsVerificationThreshold)
     )
   }
-
+  if (global.hardLimitVerificationActive && _.isNumber(global.hardLimitVerificationThreshold)) {
```
Comment (chaotixkilla) [inline lib/config-migration.js:420]: Isn't checking for `global.hardLimitVerificationActive` redundant?

**Review by chaotixkilla — CHANGES_REQUESTED** [review]
Just a small detail to simplify a condition

File: lib/config-migration.js (hunk 1 above)
Comment (josepfo) [reply lib/config-migration.js:420]: I'm not sure about this, but I believe it's possible that on 7.4 admin the hardLimitVerificationActive is disabled (false), and there's a threshold somewhere in the config, for example, if the hardLimit was ever enabled.

Comment (chaotixkilla) [reply lib/config-migration.js:420]: Ah sorry, I read the variable wrong. You're correct!

**Review by chaotixkilla — APPROVED** [review]

File: lib/config-migration.js [hunk 2]
```diff
@@ -344,24 +344,28 @@ function migrateComplianceTriggers (config) {
...
-      cashDirection: 'both',
+      direction: 'both',
+      suspensionDays: null,
       threshold,
       thresholdDays: 1,
       triggerType: triggerTypes.volume,
       requirement
     }
+    if (requirement === 'suspend') triggerConfig.suspensionDays = 1
+    return triggerConfig
```
Comment (RafaelTaranto) [inline lib/config-migration.js]: Better to add a new param and passDown suspensionDays from that

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/config-migration.js [hunk 3]
```diff
@@ -344,24 +347,29 @@ function migrateComplianceTriggers (config) {
...
-    threshold
+    threshold,
+    suspensionDays = null
   ) {
-    return {
+    const triggerConfig = {
       id: uuid.v4(),
-      cashDirection: 'both',
+      direction: 'both',
+      suspensionDays: suspensionDays,
```
Comment (RafaelTaranto) [inline lib/config-migration.js]: Ok, I still have one nitpick here. We'll be creating suspensionDays for all triggers but that should be done only for suspend. 

```js
if (!triggerType === 'suspend) return triggerConfig

return _.assign(triggerConfig, { suspensionDays })
```

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**